hikari/impl/rest.py

```diff
@@ -1211,10 +1224,19 @@ async def create_message(
         if attachments is not undefined.UNDEFINED:
             final_attachments.extend([files.ensure_resource(a) for a in attachments])
 
+        serialized_embeds: data_binding.JSONArray = []
+
         if embed is not undefined.UNDEFINED:
             embed_payload, embed_attachments = self._entity_factory.serialize_embed(embed)
-            body.put("embed", embed_payload)
             final_attachments.extend(embed_attachments)
+            serialized_embeds.append(embed_payload)
+        elif embeds is not undefined.UNDEFINED:
+            for embed in embeds:
+                embed_payload, embed_attachments = self._entity_factory.serialize_embed(embed)
+                final_attachments.extend(embed_attachments)
+                serialized_embeds.append(embed_payload)
+
+        body.put("embeds", serialized_embeds)
```

Comment:
```suggestion
        if serialized_embeds:
            body.put("embeds", serialized_embeds)
```

Reply:
What's the point of that check

Reply:
Not sending unnecessary information when not needed

Reply:
I still don't see what the issue is here. It works and it's what we've been doing the whole time. This isn't meant to be a refactoor; this is all just copy-pasted code.